lib/stk500v1.js

```diff
@@ -61,13 +61,8 @@ Stk500v1.prototype._reset = function(callback) {
   _this.connection._setDTR(false, 250, function(error) {
     if (error) { return callback(error); }
 
-    _this.connection._setDTR(true, 50, function(error) {
-      if (!error) {
-        _this.debug('reset complete.');
-      }
-
-      return callback(error);
-    });
```

Comment:
Just realised this section undoes the work in the following commit: 9a2bc68a2cb888c2ed260246a69ae88bb62dcfcd 

path to resolution: only cycle in this way for stk500v1 and ignore this for avr109.